currency-rates/internal/services/rate.go

```diff
@@ -27,7 +27,8 @@ func (r Rate) GetDollarRate(ctx context.Context) (float64, error) {
 	var rate float64
 	var err error
 	for _, c := range r.rateClients {
-		ctx, _ := context.WithTimeout(ctx, 500 * time.Millisecond)
+		ctx, cancel := context.WithTimeout(ctx, 500 * time.Millisecond)
```

Comment:
Good example of providing `magic numbers ` linter =)

So as a result you should move numbers into constants.

Reply:
done